src/components/Property/PropertyLabel.tsx

```diff
@@ -31,11 +31,10 @@ export function PropertyLabel({ name, description, uri, readOnly = false }: Prop
           )}
         </>
       }
-      description={
-        <>
-          {description}
-          {uri && <CopyUriButton uri={uri} />}
-        </>
+      info={
+        description ? (
+          <PropertyDescription uri={uri} description={description} />
```

Comment:
Maybe here we want to display the info box just to show the uri even if there is no description?